Decode/PerfItemValue.cs

```diff
@@ -10,58 +10,115 @@ namespace Microsoft.LinuxTracepoints.Decode
     using Text = System.Text;
 
     /// <summary>
-    /// Provides access to the type and content of a perf event item. An item is a field of
-    /// the event or an element of an array field of the event. The item may represent
-    /// one of the following, determined by the context that produced this PerfItemValue:
+    /// Provides access to the metadata
```

Comment:
Note: Doc comment updated, similar to the update for PerfItemMetadata. (In fact, this was and still is a copy-paste of the PerfItemMetadata comment, with a few extra lines to describe the behavior of the `Bytes` property.